packages/lwc-language-server/src/__tests__/typing-indexer.test.ts

```diff
@@ -6,6 +6,16 @@ const typingIndexer: TypingIndexer = new TypingIndexer({
     workspaceRoot: path.resolve('..', '..', 'test-workspaces', 'sfdx-workspace'),
 });
 
+// This is required in order to spyOn fsExtra functions in newer versions of
+// jest and ts-jest. Solution adapted from Jest docs here:
+// https://jestjs.io/docs/jest-object
+jest.mock('fs-extra', () => {
+  return {
+    __esModule: true,
+    ...jest.requireActual('fs-extra'),
+  };
+});
+
```

Comment:
The test `it('should not create a customlabels typing file when a project has no custom labels')` was broken without this fix. It seemed as though TypeScript changed the way it compiled ES modules, using `Object.defineProperty` for each export, which interferes with Jest's mocking. Fortunately, the fix is relatively straightforward.